src/renderers/renderer.util.tsx

```diff
@@ -155,5 +158,6 @@ export const mapStateToControlProps = (state, ownProps) => {
     enabled,
     id,
     path,
+    maxLength,
```

Comment:
This is only relevant to input controls (in contrast to say, `enabled`), so I'd try to avoid having an additional property here. Instead, the controls which are able to handle a `maxLength` property should read it within their respective `render` method.